content/3_Silver/Prefix_Sums.mdx

```diff
@@ -293,6 +293,19 @@ for i in range(Q):
 
 </LanguageSection>
 
+## Difference Array
+
+<FocusProblem problem="haybale-stacking" />
+
+<Spoiler title = "Hint">
+What if you did prefix sums, but in reverse?
```

Comment:
what

Reply:
wait yeah this doesn't make a lot of sense 😭